eclass/java-osgi.eclass

```diff
@@ -52,7 +52,7 @@ _canonicalise() {
 #	_java-osgi_plugin "JSch"
 # @CODE
 #
-# @param $1 - bundle name
```

Comment:
Why not just delete this and fix `@USAGE` instead above?

Reply:
Ditto the rest.

Reply:
> Why not just delete this and fix `@USAGE` instead above?

Coming back here later